python/bin/print_contents.py

```diff
@@ -51,14 +53,14 @@ def print_message(header, contents, one_line=False):
         '-s', '--summary', action='store_true',
         help="Print a summary of the messages in the file.")
     parser.add_argument(
-        '--time', type=str, metavar='[START][:END]',
+        '-m', '--message-type', type=str, action='append',
+        help="An optional list of class names corresponding with the message types to be displayed. "
```

Comment:
I may be misremembering how this argparse works, but isn't the usage:
`-m MSG1 -m MSG2` ? Usually when I hear a CLI variable described as a list I assume it means `-m MSG1,MSG2`

Reply:
Yes that's correct. I may actually just make it support both for convenience.